README.rst

```diff
@@ -1,51 +1,17 @@
+.. image:: https://img.shields.io/pypi/v/secrets-manager.svg
+    :target: https://pypi.org/project/secrets-manager/
 
-Steps to create repo:
-   - Log in as Curi-Bio-CI
-   - Select python-github-template as template
-   - Check box that says `include all branches`
-   - Set repo to public
-   - Publish repo
-   - To stop error messages about `master` and `development` branches not sharing any history, clone the repo, checkout development and run `git rebase -i origin/master` then `git push -f`
-   - In Actions -> Dev: click Run workflow. Wait until workflow finishes
-   - In Settings -> Security & analysis: enable Dependabot security updates
-   - In Setting -> Options, under Merge Button:
-      - Make sure "automatically delete head branches" is checked
-      - Make sure squash merging" is NOT checked
-   - In Settings -> Branches:
-      - Add Rule with master specified as Branch pattern name
-         - check Require pull requests reviews before merging
-         - check Dismiss stale pull requests
-         - check Require Review from Code Owners
-         - check Require status checks before merging
-         - Under status checks, check all of the python checks (6 total)
-         - check Include administrators
-         - check Restrict who can push to matching branches
-      - Add Rule with development specified as Branch pattern name
-         - repeat master branch steps
+.. image:: https://pepy.tech/badge/secrets-manager
+  :target: https://pepy.tech/project/secrets-manager
 
-.. image:: https://img.shields.io/pypi/v/python-github-template.svg
-    :target: https://pypi.org/project/python-github-template/
+.. image:: https://img.shields.io/pypi/pyversions/secrets-manager.svg
+    :target: https://pypi.org/project/secrets-manager/
 
-.. image:: https://pepy.tech/badge/python-github-template
-  :target: https://pepy.tech/project/python-github-template
-
-.. image:: https://img.shields.io/pypi/pyversions/python-github-template.svg
-    :target: https://pypi.org/project/python-github-template/
-
-.. image:: https://github.com/CuriBio/python-github-template/workflows/Dev/badge.svg?branch=development
+.. image:: https://github.com/CuriBio/secrets-manager/workflows/Dev/badge.svg?branch=development
    :alt: Development Branch Build
 
-.. image:: https://codecov.io/gh/CuriBio/python-github-template/branch/development/graph/badge.svg
-  :target: https://codecov.io/gh/CuriBio/python-github-template
-
-..
-   If this library uses readthedocs then put that badge here
-   .. image:: https://readthedocs.org/projects/python-github-template/badge/?version=latest
-     :target: https://python-github-template.readthedocs.io/en/latest/?badge=latest
-     :alt: Documentation Status
-
+.. image:: https://codecov.io/gh/CuriBio/secrets-manager/branch/development/graph/badge.svg
+  :target: https://codecov.io/gh/CuriBio/secrets-manager
```

Comment:
please add to the readme the need to create that IAM profile and set up those 3 secrets within github